frontend/src/Footer.tsx

```diff
@@ -96,9 +97,7 @@ function Logos() {
       <Grid item className={styles.LogosLeft}>
         <Button
           className={styles.ImageButton}
-          onClick={() =>
-            (window.location.href = "https://healthequitytracker.org/")
-          }
+          href="https://healthequitytracker.org/"
```

Comment:
What's going on here? This hard-coding seems wrong for a few reasons:
1) it won't work for the dev site or deploy previews
2) I believe linking to full url causes a full reload.

Can this just be a ReactRouterLinkButton that links to "/"?

Reply:
already updated this